torch/ao/quantization/pt2e/prepare.py

```diff
@@ -346,6 +346,7 @@ def _maybe_insert_input_observers_for_node(
     assert (
         node.target == torch.ops.aten.clone.default or
         node.target == torch.ops.aten.zeros_like.default or
+        node.target == torch.ops.aten.gelu.default or
```

Comment:
Can we also add the comment for why adding `gelu` here? Refer to https://github.com/pytorch/pytorch/blob/7963aaac4157c961c1d54033d1f2515d0c987360/torch/ao/quantization/pt2e/prepare.py#L344-L345

Reply:
May I know why we want to add observer before gelu?